<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoVisualiser</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QSlider" name="zRotate">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QSlider" name="xRotate">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="2">
     <layout class="QGridLayout" name="connectionsMenu">
      <item row="0" column="0">
       <widget class="QComboBox" name="pointOne">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="connectPoints">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Connect Points</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="pointTwo">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="disconnectPoints">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Disconnect Points</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QListWidget" name="connectionsList">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="picOutput">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>700</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="2">
     <layout class="QGridLayout" name="inputMenu" columnstretch="0,0">
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="delButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="addButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="pointInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1999999</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>A 100,100,100</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QListWidget" name="pointList"/>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="changeX">
          <property name="minimum">
           <number>-350</number>
          </property>
          <property name="maximum">
           <number>350</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="changeY">
          <property name="minimum">
           <number>-350</number>
          </property>
          <property name="maximum">
           <number>350</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="changeZ">
          <property name="minimum">
           <number>-350</number>
          </property>
          <property name="maximum">
           <number>350</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="endChanging">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QSlider" name="yRotate">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="7">
     <layout class="QHBoxLayout" name="specialMenu">
      <item>
       <widget class="QCheckBox" name="freezeSystem">
        <property name="text">
         <string>Freeze System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadFigureButton">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Load Figure...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFigureButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save Figure...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFile">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
